src/go/k8s/tests/e2e-v2/upgrade-values-check/02-delete-redpandas.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+delete:
+  - apiVersion: cluster.redpanda.com/v1alpha1
+    kind: Redpanda
+    name: redpanda
+  - apiVersion: apps/v1
+    kind: Deployment
+    name: redpanda-console
```

Comment:
Why deleting deployment if helm uninstall should remove it, right?

Reply:
Yep, this is me making checking it does delete console, i can remove it, no harm.